client/js/lounge.js

```diff
@@ -1039,9 +1042,7 @@ $(function() {
 			socket.emit("names", {target: self.data("id")});
 		}
 
-		if (screen.width > 768 && chan.hasClass("chan")) {
-			input.focus();
-		}
+		focus();
```

Comment:
I think keeping it open when clicking send and clicking the cycle button should both keep the keyboard open (or, technically reopen). I think that's most expected.